src/lib/Pjax/PjaxResponseMainContentMapper.php

```diff
@@ -54,7 +54,7 @@ public function map($response)
         $this->app->setConfig([
             'title' => $title,
             'toolbars' => ['discovery' => 1],
-            'mainContent' => ['result' => $content]
+            'mainContent' => ['result' => "<ez-serverside-view>" . $content . "</ez-serverside-view>"]
```

Comment:
on the tag name:
it is inspired by the PlatformUI `eZ.ServerSideView`, the view class used to handle pjax pages. But indeed, here that does not feel right. Maybe `<ez-server-side-content>` would be better ?

Also, since you start wrapping PJAX content with this tag, you should do the same for the Dashboard (dashboard.html.twig) and in locationview.html.twig where `<div class="ez-content-view">` becomes `<ez-serverside-view>` (until we need to specialize the behavior on those pages). With this last change, it would be nice
* to rename src/bundle/Resources/public/webcomponents/ez-main-content.html to src/bundle/Resources/public/webcomponents/ez-server-side-content.html (or which ever name we choose for the tag) and to update it to register the correct custom element
* and to update structure.css to not reference `ez-main-content`